packages/web/src/lib/txn.ts

```diff
@@ -22,6 +22,19 @@ export function getFromAddress (execParams: ExecParams): AccountAddress {
   return execParams.fromAccountAddr;
 }
 
+/**
+ * Using flatFee, if flatFee is true, set totalFee
+ * https://developer.algorand.org/tutorials/setting-transaction-fee-javascript/
+ * @param params Transaction parameters
+ * @param tx Transaction object
+ */
+export function updateFee (params: TxParams, tx: Transaction): Transaction {
```

Comment:
Can we rename `updateFee` to `updateTxFee` ?

Reply:
yes